src/Orleans.Runtime/Placement/PlacementService.cs

```diff
@@ -16,6 +17,7 @@ namespace Orleans.Runtime.Placement
     /// </summary>
     internal class PlacementService : IPlacementContext
     {
+        private const int PlacementWorkerCount = 16;
```

Comment:
Should this be configurable, or at least based on some core count?

Reply:
I suppose we could, but I'm not sure we want to expose it as a tuning knob and I'm not sure a higher or lower value is any better than this effectively random value.